docs/product/components/toggle-switch.html

```diff
@@ -0,0 +1,249 @@
+        <input type="radio" name="group" id="three">
+        <label for="three">3 hrs</label>
+        <input type="radio" name="group" id="four" checked>
+        <label for="four" class="s-toggle-switch--label-off">Off</label>
+    </div>
+    <label class="s-label">…</label>
+</div>
+    {% endhighlight %}
+        <div class="stacks-preview--example">
+            <div class="grid ai-center">
+                <div class="s-toggle-switch mr8">
+                    <input type="radio" name="example2" id="example2-w" value="Weekly">
+                    <label for="example2-w">Weekly</label>
+                    <input type="radio" name="example2" id="example2-d" value="Daily">
+                    <label for="example2-d">Daily</label>
+                    <input type="radio" name="example2" id="example2-3" value="3 hrs">
+                    <label for="example2-3">3 hrs</label>
+                    <input type="radio" name="example2" id="example2-off" checked value="Off">
+                    <label for="example2-off" class="s-toggle-switch--label-off">Off</label>
+                </div>
+                <label class="s-label">Email frequency</label>
+            </div>
+            <div class="grid ai-center mt12">
+                <div class="s-toggle-switch mr8">
+                    <input type="radio" name="example3" id="example3-a" value="Automatic">
+                    <label for="example3-a">Automatic</label>
+                    <input type="radio" name="example3" id="example3-m" value="Manual">
+                    <label for="example3-m">Manual</label>
+                    <input type="radio" name="example3" id="example3-off" checked value="Off">
+                    <label for="example3-off" class="s-toggle-switch--label-off">Off</label>
+                </div>
+                <label class="s-label">Encryption</label>
+            </div>
+        </div>
+    </div>
+</section>
+<section class="stacks-section">
+    {% header "h2", "Guidelines" %}
+    {% header "h3", "Apply the change immediately" %}
+    <p class="stacks-copy">Toggles should apply the result immediately and confirm it with <a href="/product/components/notices/#classes">a success notice</a>. Toggle switches should never require users to press a “Save” button to apply the change. When you require users to press a submit button with a toggle, you may confuse them because it’s not the expected next step. In those cases, use checkboxes instead.</p>
+    <table class="s-table s-table__lg fs-body2 va-top w100 mb24">
+        <thead>
+            <th class="btw0 bxw0 bbw3 bbw3 bc-black-3 bg-transparent fs-body1 s-table--cell3">Context</th>
+            <th class="btw0 bxw0 bbw3 bbw3 bc-red-2 fc-red-500 bg-transparent fs-body1">{% icon "Clear", "mtn1 mbn1" %} Don’t</th>
+            <th class="btw0 bxw0 bbw3 bc-green-2 fc-green-500 bg-transparent fs-body1">{% icon "Checkmark", "mtn1 mbn1" %} Do</th>
+        </thead>
+        <tbody>
```

Comment:
This table is trolling our responsive layout. Let's wrap it like the other ones in `overflow-x-auto`.